CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 
 * [#7860](https://github.com/rubocop-hq/rubocop/issues/7860): Change `AllowInHeredoc` option of `Layout/TrailingWhitespace` to `true` by default. ([@koic][])
 * [#7094](https://github.com/rubocop-hq/rubocop/issues/7094): Clarify alignment in `Layout/MultilineOperationIndentation`. ([@jonas054][])
+* [#4245](https://github.com/rubocop-hq/rubocop/issues/4245): **(Breaking)** Inspect all files given on command line unless `--only-recognized-file-types` is given. ([@jonas054][])
```

Comment:
Shouldn't this be #7930 and not the original issue number which is the 4245?

Reply:
No, if there is an issue that is fixed by the PR we usually refer to the issue. At least that's what I've always done. :)

You'll be able to find the PR through a link at the bottom of the issue, even though I made a mistake and forgot the hash mark before the issue number in the commit message.